index.d.ts

```diff
@@ -0,0 +1,104 @@
+// TypeScript Version: 3.7
+
+import {Test} from 'unist-util-is'
+import * as unist from 'unist'
+
+export {}
+
+type Null = undefined | null
```

Comment:
I don't think we need this.
Rather than using Null, we can just do like the below example.

```ts
someStrKey?: string | null
```

Reply:
But don't fix this now. I think we need to discuss https://github.com/unifiedjs/unified-message-control/pull/7#discussion_r368238256  first.